app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt

```diff
@@ -459,14 +459,23 @@ class PlayerFragment : BaseFragment(), OnlinePlayerOptions {
     }
 
     private fun toggleDescription() {
+        var viewInfo = if (!isLive) TextUtils.SEPARATOR + streams.uploadDate else ""
         if (binding.descLinLayout.isVisible) {
             // hide the description and chapters
             binding.playerDescriptionArrow.animate().rotation(0F).setDuration(250).start()
             binding.descLinLayout.visibility = View.GONE
+
+            // show formated short view count
+            viewInfo = getString(R.string.views, streams.views.formatShort()) + viewInfo
+            binding.playerViewsInfo.text = viewInfo
```

Comment:
We could even move that line below the if/else condition because it's used in both the if and the else condition now :)

Reply:
Yep, didn't noticed :)